apps/src/code-studio/pd/workshop_dashboard/attendance/workshop_attendance.jsx

```diff
@@ -151,7 +151,7 @@ const WorkshopAttendance = React.createClass({
   },
 
   renderAdminControls() {
-    if (this.shouldUseNewAttendance || !this.state.accountRequiredForAttendance || !this.isAdmin) {
+    if (this.shouldUseNewAttendance || !this.state.accountRequiredForAttendance || !this.permission.isWorkshopAdmin) {
```

Comment:
Also incidentally these were missing `permission` in the original PR, so they will always be false. Luckily that doesn't break any functionality for our organizers and facilitators, but it does break workshop admin functionality.